ui/src/App/index.jsx

```diff
@@ -69,6 +69,7 @@ export default class App extends Component {
 
   httpLink = new HttpLink({
     uri: process.env.GRAPHQL_ENDPOINT,
+    credentials: 'same-origin',
```

Comment:
From https://www.apollographql.com/docs/react/recipes/authentication/#cookie:

>  You just need to pass the credentials option. e.g. credentials: 'same-origin' as shown below, if your backend server is the same domain or else credentials: 'include' if your backend is a different domain.

Because we're using a proxy to talk to web-server, I figured we can use use `same-origin`. @djmitche Does that make sense to you? or do you think it should be set to `include`.

Reply:
Ugh, for the herokuapp domain it will need to be `include`, since it's the one case where it is *not* the same origin.  Maybe just add a special case here for `rootUrl == 'https://taskcluster.net'`?

Reply:
@djmitche Just to clarify, are you suggesting we check the domains of the current URL and `rootUrl`?

Something like `window.location.hostname === new URL(process.env.TASKCLUSTER_ROOT_URL).hostname ? 'same-origin' : 'include'` should work, I think.

Reply:
I think you can probably just do `process.env.TASKCLUSTER_ROOT_URL === 'https://taskcluster.net' ? 'include' : 'same-origin'`.

Reply:
@helfi92 Interesting. What other `TASKCLUSTER_ROOT_URL` do we use in the builds?

Reply:
The site can be deployed by anyone using their own configuration variables. Examples:
* https://taskcluster.imbstack.com/
* https://docs.taskcluster.net/docs
* https://taskcluster-ui.herokuapp.com/

Reply:
Gotcha.